browser/tor/brave_tor_browsertest.cc

```diff
@@ -346,25 +348,53 @@ IN_PROC_BROWSER_TEST_F(BraveTorTest, ResetBridges) {
   EXPECT_FALSE(CheckComponentExists(tor::kTorPluggableTransportComponentId));
 }
 
-class BraveTorTest_EnableTorHttpsOnlyFlag : public BraveTorTest {
+class BraveTorTest_EnableTorHttpsOnlyFlag
+    : public BraveTorTest,
+      public ::testing::WithParamInterface<bool> {
  public:
   BraveTorTest_EnableTorHttpsOnlyFlag() {
-    feature_list_.InitAndEnableFeature(
-        blink::features::kBraveTorWindowsHttpsOnly);
+    if (IsBraveHttpsByDefaultEnabled()) {
+      std::vector<base::test::FeatureRef> enabled_features{
+          net::features::kBraveTorWindowsHttpsOnly,
+          net::features::kBraveHttpsByDefault};
+      std::vector<base::test::FeatureRef> disabled_features{/* none */};
+      scoped_feature_list_.InitWithFeatures(enabled_features,
+                                            disabled_features);
+    } else {
+      scoped_feature_list_.InitWithFeatures(
+          {net::features::kBraveTorWindowsHttpsOnly} /* enabled */,
+          {net::features::kBraveHttpsByDefault} /* disabled */
+      );
+    }
```

Comment:
this is what I meant:
```
std::vector<base::test::FeatureRef> enabled_features{
    net::features::kBraveTorWindowsHttpsOnly};
std::vector<base::test::FeatureRef> disabled_features;

if (IsBraveHttpsByDefaultEnabled()) {
  enabled_features.push_back(net::features::kBraveHttpsByDefault);
} else {
  disabled_features.push_back(net::features::kBraveHttpsByDefault);
}

scoped_feature_list_.InitWithFeatures(enabled_features, disabled_features);
```
you should have only one `scoped_feature_list_.InitWithFeatures` call.

Reply:
Got it! Fixed